src/ci_providers/provider_circleci.js

```diff
@@ -24,12 +24,7 @@ function _getBranch (inputs) {
 
 function _getSHA (inputs) {
   const { args, envs } = inputs
-  try {
-    const sha = envs.CIRCLE_SHA1
-    return args.sha || sha
-  } catch (error) {
-    throw new Error(`There was an error getting the commit SHA: ${error}`)
-  }
+  return args.sha || envs.CIRCLE_SHA1
```

Comment:
What happens in the case both are empty?

Reply:
has not been an issue on the bash uploader, assuming it's a standard env variable. in any case, it will return `undefined`